docs/building_illixr.md

```diff
@@ -1,153 +1,199 @@
 # Building ILLIXR
 
-## Basic usage
+The ILLIXR application is kick-started through a tool called [_Runner_][10]
+    (found in `runner/` and `runner.sh`).
+The Runner tool is responsible for
+    preparing the environment,
+    downloading required assets/code,
+    compiling each plugin,
+    and
+    launching the ILLIXR application.
+Runner is necessary for our project, since ILLIXR manages plugins and data that span
```

Comment:
is the comma necessary? (nitpicky detail)

Reply:
No, but it's quite [common in American English](https://en.wikipedia.org/wiki/Serial_comma).

Reply:
Or is this about the comma before "since"?

Reply:
yes